### ISSUE TITLE
Windows could not start the Red5 on local computer

### ISSUE THREAD

**Issue by davorgit** [Red5/red5-server#5]
fresh install on win7 64bit ultimate with red5 ver. 1.0.2 and java7 32bit:
![servicered5](https://cloud.githubusercontent.com/assets/1269067/2716961/beaaea62-c535-11e3-9109-6009623b4c6b.jpg)

**Comment by mariapanjaitan** [comment]
can u tell me how to fix this error please? I can't start my red5 too and I had the same issue with davorgit.. how can I fix this issue? sorry if I ask u again, but I don't understand your explanation above,..  your explanation would be a great help..

**Comment by mariapanjaitan** [comment]
t2y... can u help me some help according to this issue please??? I need your help to fix this error sir..

**Comment by mondain** [comment]
The red5 windows service app is fixed in the 1.0.8-M1+ series.
